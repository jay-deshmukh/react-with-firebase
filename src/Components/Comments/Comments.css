.back {
  font-size: 25px
}

.modal-head {
  background-color: #1e3799 !important; 
  color: white; 
}
.modal-card-title {
  color: white !important;
  text-align: left;
  padding-left: 20px;
  text-decoration: none;
}
.profile-image{
  padding-left: unset !important;
  margin-left: unset !important;
  width: auto !important;
}
.profile-name {
  padding-top: 20px;
  background-color: #1e3799;
  margin: 2px;
  text-align: left;
  margin-top: 2%;
  margin-bottom: 2%;
  margin-right: 10%;
  color : white;
}
#comment-input {
  width: 100%;
  padding: 5%;
}

#send {
  margin-left: -10%;
}

.plane  {
  margin-top: 20%;
  background-color: #1e3799;
  color: white;
  font-size: 25px;
}
Link {
  color: white !important;
}