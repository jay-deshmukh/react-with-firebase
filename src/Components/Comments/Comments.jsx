import React, { Component } from 'react';
import './Comments.css'
class Comments extends Component {

  hideModal = () => {
    const element = document.getElementById('commentsModal')
    element.classList.remove('is-active');
  }
  render() {
    return (
      <div className="container">
        <div className="modal" id="commentsModal">
          <div className="modal-background"></div>
          <div className="modal-card">
            <header className="modal-card-head modal-head">
              <p className="fas fa-arrow-left back"></p>
              <p onClick={this.hideModal} className="modal-card-title">{"Movie"}  Coments</p>
            </header>
            <section className="modal-card-body">
            <div className="columns is-mobile">
              <div className="column is-one-fifth profile-image">
                <figure className="image is-48x48 ">
                  <img alt="profile" className="is-rounded" src="https://bulma.io/images/placeholders/128x128.png" />
                </figure>
              </div>
                <div className="column profile-name">
                <p>Alpha</p>
              </div>
					  </div>
            <div className="columns is-mobile">
              <div className="column is-one-fifth profile-image">
                <figure className="image is-48x48 ">
                  <img alt="profile" className="is-rounded" src="https://bulma.io/images/placeholders/128x128.png" />
                </figure>
              </div>
                <div className="column profile-name">
                <p>Bravo</p>
              </div>
					  </div>
            <div className="columns is-mobile">
              <div className="column is-one-fifth profile-image profile-image">
                <figure className="image is-48x48">
                  <img alt="profile" className="is-rounded" src="https://bulma.io/images/placeholders/128x128.png" />
                </figure>
              </div>
                <div className="column profile-name">
                <p>Charlie</p>
              </div>
					  </div>
            <div className="columns is-mobile">
              <div className="column is-one-fifth profile-image">
                <figure className="image is-48x48 ">
                  <img alt="profile" className="is-rounded" src="https://bulma.io/images/placeholders/128x128.png" />
                </figure>
              </div>
                <div className="column profile-name">
                <p>Delta</p>
              </div>
					  </div>
            <div className="columns is-mobile">
              <div className="column is-one-fifth profile-image">
                <figure className="image is-48x48 ">
                  <img alt="profile" className="is-rounded" src="https://bulma.io/images/placeholders/128x128.png" />
                </figure>
              </div>
                <div className="column profile-name">
                <p>Elexss</p>
              </div>
					  </div>
            </section>
            <footer className="modal-card-foot">
              <input className="input" placeholder="The Movie Is So Cool" id="comment-input"/>
              <figure className="image is-48x48" id="send">
                  <img alt="profile" className="is-rounded" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ" />
              </figure>
            </footer>
          </div>
        </div>
      </div>
    );
  }
}

export default Comments;