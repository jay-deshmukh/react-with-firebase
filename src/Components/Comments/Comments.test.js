import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Comments from './Comments';
describe('<Comments />' , () => {
  it('Renders Comments' , ()=>{
    const wrapper = shallow(<Comments />);
    expect(wrapper).toMatchSnapshot();
  })
});
