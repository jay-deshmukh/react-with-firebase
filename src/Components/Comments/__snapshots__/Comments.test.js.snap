// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Comments /> Renders Comments 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Comments />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="modal is-active"
        id="commentsModal"
      >
        <div
          className="modal-background"
        />
        <div
          className="modal-card"
        >
          <header
            className="modal-card-head modal-head"
          >
            <Link
              replace={false}
              to="/"
            >
              <i
                className="fas fa-arrow-left back"
              />
            </Link>
            <p
              className="modal-card-title"
            >
               Comments
            </p>
          </header>
          <section
            className="modal-card-body"
          />
          <footer
            className="modal-card-foot"
          >
            <input
              className="input"
              id="comment-input"
              onChange={[Function]}
              placeholder="The Movie Is So Cool"
            />
            <figure
              className="image is-48x48"
              id="send"
            >
              <img
                alt="profile"
                className="is-rounded"
                onClick={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
              />
            </figure>
          </footer>
        </div>
      </div>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="modal-background"
          />,
          <div
            className="modal-card"
          >
            <header
              className="modal-card-head modal-head"
            >
              <Link
                replace={false}
                to="/"
              >
                <i
                  className="fas fa-arrow-left back"
                />
              </Link>
              <p
                className="modal-card-title"
              >
                 Comments
              </p>
            </header>
            <section
              className="modal-card-body"
            />
            <footer
              className="modal-card-foot"
            >
              <input
                className="input"
                id="comment-input"
                onChange={[Function]}
                placeholder="The Movie Is So Cool"
              />
              <figure
                className="image is-48x48"
                id="send"
              >
                <img
                  alt="profile"
                  className="is-rounded"
                  onClick={[Function]}
                  src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                />
              </figure>
            </footer>
          </div>,
        ],
        "className": "modal is-active",
        "id": "commentsModal",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "modal-background",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <header
                className="modal-card-head modal-head"
              >
                <Link
                  replace={false}
                  to="/"
                >
                  <i
                    className="fas fa-arrow-left back"
                  />
                </Link>
                <p
                  className="modal-card-title"
                >
                   Comments
                </p>
              </header>,
              <section
                className="modal-card-body"
              />,
              <footer
                className="modal-card-foot"
              >
                <input
                  className="input"
                  id="comment-input"
                  onChange={[Function]}
                  placeholder="The Movie Is So Cool"
                />
                <figure
                  className="image is-48x48"
                  id="send"
                >
                  <img
                    alt="profile"
                    className="is-rounded"
                    onClick={[Function]}
                    src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                  />
                </figure>
              </footer>,
            ],
            "className": "modal-card",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Link
                    replace={false}
                    to="/"
                  >
                    <i
                      className="fas fa-arrow-left back"
                    />
                  </Link>,
                  <p
                    className="modal-card-title"
                  >
                     Comments
                  </p>,
                ],
                "className": "modal-card-head modal-head",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <i
                      className="fas fa-arrow-left back"
                    />,
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fas fa-arrow-left back",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      undefined,
                      " Comments",
                    ],
                    "className": "modal-card-title",
                    "onClick": undefined,
                  },
                  "ref": null,
                  "rendered": Array [
                    undefined,
                    " Comments",
                  ],
                  "type": "p",
                },
              ],
              "type": "header",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [],
                "className": "modal-card-body",
              },
              "ref": null,
              "rendered": Array [],
              "type": "section",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <input
                    className="input"
                    id="comment-input"
                    onChange={[Function]}
                    placeholder="The Movie Is So Cool"
                  />,
                  <figure
                    className="image is-48x48"
                    id="send"
                  >
                    <img
                      alt="profile"
                      className="is-rounded"
                      onClick={[Function]}
                      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                    />
                  </figure>,
                ],
                "className": "modal-card-foot",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "input",
                    "id": "comment-input",
                    "onChange": [Function],
                    "placeholder": "The Movie Is So Cool",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "input",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <img
                      alt="profile"
                      className="is-rounded"
                      onClick={[Function]}
                      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                    />,
                    "className": "image is-48x48",
                    "id": "send",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "profile",
                      "className": "is-rounded",
                      "onClick": [Function],
                      "src": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  "type": "figure",
                },
              ],
              "type": "footer",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="modal is-active"
          id="commentsModal"
        >
          <div
            className="modal-background"
          />
          <div
            className="modal-card"
          >
            <header
              className="modal-card-head modal-head"
            >
              <Link
                replace={false}
                to="/"
              >
                <i
                  className="fas fa-arrow-left back"
                />
              </Link>
              <p
                className="modal-card-title"
              >
                 Comments
              </p>
            </header>
            <section
              className="modal-card-body"
            />
            <footer
              className="modal-card-foot"
            >
              <input
                className="input"
                id="comment-input"
                onChange={[Function]}
                placeholder="The Movie Is So Cool"
              />
              <figure
                className="image is-48x48"
                id="send"
              >
                <img
                  alt="profile"
                  className="is-rounded"
                  onClick={[Function]}
                  src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                />
              </figure>
            </footer>
          </div>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="modal-background"
            />,
            <div
              className="modal-card"
            >
              <header
                className="modal-card-head modal-head"
              >
                <Link
                  replace={false}
                  to="/"
                >
                  <i
                    className="fas fa-arrow-left back"
                  />
                </Link>
                <p
                  className="modal-card-title"
                >
                   Comments
                </p>
              </header>
              <section
                className="modal-card-body"
              />
              <footer
                className="modal-card-foot"
              >
                <input
                  className="input"
                  id="comment-input"
                  onChange={[Function]}
                  placeholder="The Movie Is So Cool"
                />
                <figure
                  className="image is-48x48"
                  id="send"
                >
                  <img
                    alt="profile"
                    className="is-rounded"
                    onClick={[Function]}
                    src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                  />
                </figure>
              </footer>
            </div>,
          ],
          "className": "modal is-active",
          "id": "commentsModal",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "modal-background",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <header
                  className="modal-card-head modal-head"
                >
                  <Link
                    replace={false}
                    to="/"
                  >
                    <i
                      className="fas fa-arrow-left back"
                    />
                  </Link>
                  <p
                    className="modal-card-title"
                  >
                     Comments
                  </p>
                </header>,
                <section
                  className="modal-card-body"
                />,
                <footer
                  className="modal-card-foot"
                >
                  <input
                    className="input"
                    id="comment-input"
                    onChange={[Function]}
                    placeholder="The Movie Is So Cool"
                  />
                  <figure
                    className="image is-48x48"
                    id="send"
                  >
                    <img
                      alt="profile"
                      className="is-rounded"
                      onClick={[Function]}
                      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                    />
                  </figure>
                </footer>,
              ],
              "className": "modal-card",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <Link
                      replace={false}
                      to="/"
                    >
                      <i
                        className="fas fa-arrow-left back"
                      />
                    </Link>,
                    <p
                      className="modal-card-title"
                    >
                       Comments
                    </p>,
                  ],
                  "className": "modal-card-head modal-head",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <i
                        className="fas fa-arrow-left back"
                      />,
                      "replace": false,
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "className": "fas fa-arrow-left back",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "i",
                    },
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        undefined,
                        " Comments",
                      ],
                      "className": "modal-card-title",
                      "onClick": undefined,
                    },
                    "ref": null,
                    "rendered": Array [
                      undefined,
                      " Comments",
                    ],
                    "type": "p",
                  },
                ],
                "type": "header",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [],
                  "className": "modal-card-body",
                },
                "ref": null,
                "rendered": Array [],
                "type": "section",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <input
                      className="input"
                      id="comment-input"
                      onChange={[Function]}
                      placeholder="The Movie Is So Cool"
                    />,
                    <figure
                      className="image is-48x48"
                      id="send"
                    >
                      <img
                        alt="profile"
                        className="is-rounded"
                        onClick={[Function]}
                        src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                      />
                    </figure>,
                  ],
                  "className": "modal-card-foot",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "input",
                      "id": "comment-input",
                      "onChange": [Function],
                      "placeholder": "The Movie Is So Cool",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "input",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <img
                        alt="profile"
                        className="is-rounded"
                        onClick={[Function]}
                        src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ"
                      />,
                      "className": "image is-48x48",
                      "id": "send",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "alt": "profile",
                        "className": "is-rounded",
                        "onClick": [Function],
                        "src": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ17BsdB-CGr94PrmM0YOgSJa5prx8wbqu-7cGTNbjAzzezn9osCQ",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "img",
                    },
                    "type": "figure",
                  },
                ],
                "type": "footer",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
